### DISCUSSION TITLE
math: not supported in italic inline text

### DISCUSSION THREAD

**Discussion by matsfinsas** [community/community#17264] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/12988459/170250662-cf17e473-f3a0-4cce-a45d-8b2501889081.png)

It would be nice if math also rendered when text is formatted _italic_.

**Comment by nschloe** [comment]
Example:

```markdown
Text... $\Omega(69)$ inline

**And if we put $\Omega(69)$ inline and bold**

_And if we put $\Omega(69)$ inline and italic_
```

Text... $\Omega(69)$ inline

**And if we put $\Omega(69)$ inline and bold**

_And if we put $\Omega(69)$ inline and italic_


_Edit:_ I've added this to the math bug repo. https://github.com/nschloe/github-math-bugs#sumproduct-signs-in-inline-mode-or-fractions

**Comment by nschloe** [comment, marked as answer]
[One can now use inline math with backticks](https://github.blog/changelog/2023-05-08-new-delimiter-syntax-for-inline-mathematical-expressions/) to solve the problem:

> _Some italic text with $`\Omega`$ math._
